### PR TITLE
[TRANSLATION] translate MITmaths.md && CS126.md

### PR DESCRIPTION
**Pull request by smxm**

### CODE REVIEW

**Comment by smxm** [comment]
Is this the way to upload the translated version? I am not quite familiar with github yet.

File: docs/数学基础/MITmaths.en.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Descriptions
+
+- Offered by: MIT
+- Prerequisites: English
+- Programming Languages: None
+- Difficulty: 🌟🌟
+- Class Hour: Varying from person to person
+
+The calculus course of *MIT consists of MIT18.01: Single variable calculus* and *MIT18.02: Multi variable calculus*. If you are confident in your math, you can just read the course notes, which are written in a very simple and vivid way, so that you will not be tired of doing homework but can really see the essence of calculus.
```
Comment (PKUFlyingPig) [inline docs/数学基础/MITmaths.en.md:11]: The calculus course at MIT consists of *MIT18.01: Single Variable Calculus* and *MIT18.02: Multivariable calculus*.

File: docs/数学基础/MITmaths.en.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+In addition to the course materials, the famous Youtuber **3Blue1Brown**'s video series [The Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr) are also great learning resources.
```
Comment (PKUFlyingPig) [inline docs/数学基础/MITmaths.en.md:13]: [The Essence of Calculus](https://www.youtube.com/playlist?list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr)

File: docs/数学进阶/CS126.en.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Descriptions
+
+- Offered by: UC Berkeley
+- Prerequisites: CS70, Calculus, Linear Algebra
+- Programming Languages: Python
+- Difficulty: 🌟🌟🌟🌟🌟
+- Class Hour: 100 hours
+
+Berkeley's advanced probability course, which involves relatively advanced theoretical content such as statistics and stochastic processes, requires certain mathematical foundation. But as long as you stick with it you will certainly take your mastery of probability theory to a new level.
```
Comment (PKUFlyingPig) [inline docs/数学进阶/CS126.en.md:11]: This is Berkeley's advanced probability course, which involves relatively advanced theoretical content such as statistics and stochastic processes, so a solid mathematical foundation is required.

File: docs/数学进阶/CS126.en.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+The course is designed by Professor Jean Walrand, who has written an accompanying textbook, [Probability in Electrical Engineering and Computer Science](https://link.springer.com/book/10.1007/978-3-030-49995-2), in which each chapter uses a specific algorithm as a practical example to demonstrate the application of theory in practice. Such as PageRank, Route Planing, Speech Recognition, etc. The book is open source and can be downloaded as a free PDF or Epub version.
+
+Jean Walrand has also created accompanying Python implementations of the examples throughout the book, which are published online as  [Jupyter Notebook](https://jeanwalrand.github.io/PeecsJB/intro.html) that readers can modify, debug and run them online.
```
Comment (PKUFlyingPig) [inline docs/数学进阶/CS126.en.md:15]: there are two spaces before [Jupyter Notebook](...)

Comment (PKUFlyingPig) [reply docs/数学进阶/CS126.en.md:15]: that readers can modify, debug and run them online interactively

File: docs/数学进阶/CS126.en.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+In addition to the Homeworks, there are 9 Labs in this course which will allow you to use probability theory to solve actual problems.
```
Comment (PKUFlyingPig) [inline docs/数学进阶/CS126.en.md:17]: In addition to the Homework, nine Labs will allow you to use probability theory to solve practical problems in Python.

File: docs/数学进阶/CS126.en.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## Course Resources
+
+- Course Website: https://inst.eecs.berkeley.edu/~ee126/fa20/content.html<https://inst.eecs.berkeley.edu/~ee126/fa20/content.html>
+- Textbook: [PDF](https://link.springer.com/content/pdf/10.1007%2F978-3-030-49995-2.pdf), [Epub](https://link.springer.com/download/epub/10.1007%2F978-3-030-49995-2.epub), [Jupyter Notebook][Jupyter_Notebook]
+- Assignments: see course website for specific requirements.
```
Comment (PKUFlyingPig) [inline docs/数学进阶/CS126.en.md:23]: Assignments: refer to the course website.

File: docs/数学进阶/CS126.en.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+Jean Walrand has also created accompanying Python implementations of the examples throughout the book, which are published online as  [Jupyter Notebook](https://jeanwalrand.github.io/PeecsJB/intro.html) that readers can modify, debug and run them online.
+
+In addition to the Homeworks, there are 9 Labs in this course which will allow you to use probability theory to solve actual problems.
+
+## Course Resources
+
+- Course Website: https://inst.eecs.berkeley.edu/~ee126/fa20/content.html<https://inst.eecs.berkeley.edu/~ee126/fa20/content.html>
+- Textbook: [PDF](https://link.springer.com/content/pdf/10.1007%2F978-3-030-49995-2.pdf), [Epub](https://link.springer.com/download/epub/10.1007%2F978-3-030-49995-2.epub), [Jupyter Notebook][Jupyter_Notebook]
```
Comment (PKUFlyingPig) [inline docs/数学进阶/CS126.en.md:22]: What do you mean by "Jupyter_Notebook" ?

**Comment by PKUFlyingPig** [comment]
> Is this the way to upload the translated version? I am not quite familiar with github yet.

Yes, you did it in the exactly correct way.

**Comment by smxm** [comment]
Thanks for your review, I have pushed a new commit.

**Comment by PKUFlyingPig** [comment]
The first review section has not been resolved entirely. "MIT18.01: Single Variable Calculus and MIT18.02: Multivariable calculus"

**Comment by smxm** [comment]
Now there should be no problem, just found that the Chinese version also has a space between Multi and variable, maybe it needs to be fixed

**Comment by smxm** [comment]
Thanks again very much for the review, I am a rookie, but I hope very much to be able to make a contribution

**Comment by PKUFlyingPig** [comment]
> Thanks again very much for the review, I am a rookie, but I hope very much to be able to make a contribution

You've done a real good job ~~

**Pull request merged**